axon-framework/events/event-bus-and-event-store.md

```diff
@@ -393,14 +393,23 @@ If no explicit `eventSerializer` is configured, events are serialized using the
 
 ## Distributing Events
 
-The use of an `EventStore` provides a simple way of distributing events: by sharing the data source.
-This is the approach taken by [Axon Server](../../axon-server-introduction.md), for example.
-
-Such an approach will also suffice whenever the [`EmbeddedEventStore`](#embedded-event-store) is used.
-Doing so requires the correct configuration of each application instance to point to the same data source.
-Listing _all_ possible data sources is out of the scope of this guide however.
-The documentation for the chosen source should thus be read for specifics on this.
-
-Alternatively, you can choose other components to distribute events.
-Axon provides a couple of these as [extension modules](../../extensions) for example [Spring AMQP](../../extensions/spring-amqp.md) or [Kafka](../../extensions/kafka.md).
-Note that neither of these are intended as an [Event Store](#event-store), but **only** as a message broker.
\ No newline at end of file
+To distribute events between applications, it is paramount to know whether the applications belong to the same [bounded context](../../architecture-overview/ddd-cqrs-concepts.md#bounded-context).
+If they do, they're allowed to understand each other's language.
+In other words, they should be free to understand the events altogether.
+
+As such, we can share the `EventStore`'s data source between these applications typically suffices.
```

Comment:
```suggestion
As such, sharing the `EventStore`'s data source between these applications typically suffices.
```

Reply:
Grammarly actually pushed me to use the current form instead of just 'sharing' :sweat_smile:

Reply:
hahaha ok... then, IMO, it doesn't make much sense with the `we can share ... tipically suffices` 😢

Reply:
Ha, alright, alright, I'll rephrase it to get something more sensible :-)